subsys/mgmt/ec_host_cmd/ec_host_cmd_handler.c

```diff
@@ -97,6 +100,30 @@ enum ec_host_cmd_status ec_host_cmd_send_in_progress_status(void)
 
 	return ret;
 }
+
+enum ec_host_cmd_status ec_host_cmd_send_in_progress_continue(ec_host_cmd_in_progress_cb_t cb,
+							      void *user_data)
+{
+	if (cmd_in_progress) {
+		return EC_HOST_CMD_BUSY;
+	}
+
+	cmd_in_progress = true;
+	cb_in_progress = cb;
+	user_data_in_progress = user_data;
+	saved_status = EC_HOST_CMD_UNAVAILABLE;
+	LOG_INF("HC pending");
```

Comment:
should this be a `LOG_DBG`?

Reply:
These are two additional logs per very IN_PROGRESS command, which is very very rare one. IMO `LOG_INF` is fine.